test/configs/oneshard.txt

```diff
@@ -0,0 +1,32 @@
+127.0.0.1 9000 leader 0
```

Comment:
as you introduced new config file.

If this file will be used regularly, can you add a new deploy/test command in README.md; otherwise ignore this message.